benchmark_utils/optuna_solver.py

```diff
@@ -83,3 +87,35 @@ def get_result(self):
 
     def warmup_solver(self):
         pass
+
+
+class AvgClf(OSolver):
+    def __init__(self, estimators):
+        self.estimators = estimators
+
+    def predict(self, X_test):
+        y_pred = []
+        for e in self.estimators:
+            y_pred.append(e.predict(X_test))
+        y_pred = np.array(y_pred)
+        y_pred = stats.mode(y_pred, keepdims=True)[0][0]
+
+        return y_pred
+
+    def predict_proba(self, X_test):
+        y_pred = []
+        for e in self.estimators:
+            y_pred.append(e.predict_proba(X_test))
+        y_pred = np.array(y_pred)
+        y_pred = np.mean(y_pred, axis=0)
+
+        return y_pred
+
+    def score(self, X, y):
+        cpt = 0
+        length = y.shape
+        y_pred = self.predict(X)
+        for i in range(length[0]):
+            if y[i] == y_pred[i]:
+                cpt += 1
+        return cpt/length[0]

```

Comment:
You can simply do:
`y_pred = self.predict(X)` and then call `accuracy_score`.
Also, you could derive your class from `sklearn.base._ClassifierMixin` which implement `score` for you.